scripts/motraddr.sh

```diff
@@ -167,7 +169,7 @@ echo "# Application: All"
 echo "#"
 
 echo
-echo "HA_ENDPOINT_ADDR = ${p[0]}"
+echo "HA_ENDPOINT_ADDR = $(echo "${p[5]}" | cut -d'@' -f 1)@$(echo "${p[0]}" | cut -d'@' -f 2)"
```

Comment:
You can use bash's build-in Parameter Expansion feature to split variable value to parts. For example:

```bash
$ aa=bb@cc
$ echo ${aa%@*}@${aa#*@}
bb@cc
```

Reply:
Changed to use expansion in 5397c7c